### PR TITLE
Prepare release 0.11.0

### PR DESCRIPTION
**Pull request by gwarf**
Prepare release 0.11.0.

### CODE REVIEW

File: rpm/cloud-info-provider-opennebula.spec [hunk 1]
```diff
@@ -25,6 +23,9 @@ Install the cloud-information-provider and the Cloud Middleware dependencies.
 %files
 
 %changelog
+* Sun Jan 27 2018 Baptiste Grenier <baptiste.grenier@egi.eu> 0.11.0
```
Comment (enolfc) [inline rpm/cloud-info-provider-opennebula.spec]: 2019...

**Comment by gwarf** [comment]
Oops, done!

File: rpm/cloud-info-provider-opennebula.spec (hunk 1 above)
Comment (gwarf) [reply rpm/cloud-info-provider-opennebula.spec]: done

**Comment by enolfc** [comment]
Feel free to ignore codacy, we may turn that into optional

**Pull request merged**